examples/05_glm_second_level/plot_oasis.py

```diff
@@ -32,6 +32,12 @@
     - Gael Varoquaux, Apr 2014
 
 """
+# %%
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    raise RuntimeError("This script needs the matplotlib library")
+
```

Comment:
some examples have this where as others do not: 

I fell that if they are plotting things they should all have it (or none should have it)

Reply:
Indeed

Reply:
They should all have it.

Reply:
OK will update
- [ ] add to all examples

Reply:
OK what I have done is to add this try / except block in all examples that use `pyplot` but NOT `nilearn.plotting` because if they import from the latter they will anyway get a warning if matplotlib is not installed

Otherwise I was ending up adding this block in literally all the examples.